angular/core/components/thread_page/comment_form/attachment_form.coffee

```diff
@@ -28,6 +28,9 @@ angular.module('loomioApp').directive 'attachmentForm', ->
     $scope.failure = (response) ->
       $scope.model.setErrors(response.data.errors)
 
+    $scope.failure = (response) ->
```

Comment:
I think this might be the result of a weird merge...there are a couple of other things in here that I can't think of another explanation for